indigo_app/views/works.py

```diff
@@ -466,7 +466,15 @@ def get_context_data(self, **kwargs):
 
         actions = self.work.action_object_actions.all()
         versions = self.work.versions().all()
-        entries = sorted(chain(actions, versions), key=lambda x: x.revision.date_created if hasattr(x, 'revision') else x.timestamp, reverse=True)
+        task_actions = self.get_task_actions()
+        if task_actions:
+            entries = sorted(chain(actions, versions, task_actions),
+                             key=lambda x: x.revision.date_created if hasattr(x, 'revision') else x.timestamp,
+                             reverse=True)
+        else:
+            entries = sorted(chain(actions, versions),
+                             key=lambda x: x.revision.date_created if hasattr(x, 'revision') else x.timestamp,
+                             reverse=True)
```

Comment:
I feel like lines 470–477 can be simplified but I tried a couple of things and they didn't work